src/test/ui/unsized-locals/unsized-local-pat.rs

```diff
@@ -10,4 +10,8 @@ fn f1(box box _b: Box<Box<[u8]>>) {}
 fn f2((_x, _y): (i32, [i32])) {}
 //~^ ERROR: the size for values of type `[i32]` cannot be known at compilation time [E0277]
 
-fn main() {}
+fn main() {
+    let foo: Box<[u8]> = Box::new(*b"foo");
+    let _foo: [u8] = *foo;
+    //~^ ERROR: the size for values of type `[u8]` cannot be known at compilation time [E0277]
+}
```

Comment:
Now the file name doesn't make much sense any more though, and the comment at the top of the file.^^
```
// Ensure that even with unsized_fn_params, we do not accept unsized types
// for other locals of parameter patterns.
```

Reply:
:+1:, force pushed again.